.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,59 @@
+
+name: 🐞 Bug
+description: Report an issue to help improve the project.
+title: "[BUG] <titile>"
+labels: ["bug"]
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please serch to see if an issue already exists for the bug you encountered.
+      options:
+      - label: I have searched the existing issues
+        required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: A brief description of the question or issue, also include what you tried and what didn't work
+    validations:
+      required: true
+  - type: textarea
+    id: Reproduce
+    attributes:
+      label: To Reproduce
+      description: Steps to reproduce this bug
+    validations:
+      required: true
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Screenshots
+      description: Please add screenshots if applicable
+    validations:
+      required: false
+  - type: textarea
+    id: extrainfo
+    attributes:
+      label: Additional information
+      description: Is there anything else we should know about this bug? (like linux distribution(version + arch), python version and etc)   
```

Comment:
Punctuation and wording could use cleanup, but this is good:

"Is there anything else we should know about this issue? (Including linux distribution (version + arch), python version, etc.)"

I'm somewhat inclined to wish for pulldowns at least for version and architecture; although you have a point that it obliges us to maintain the options. (We can always mitigate that with an "other" choice, and update lazily depending on use.)

Reply:
My enthusiasm for your template idea is largely motivated by the fact that most of our defect reports have very little information. Prompting based on this sort of vague category may well help, but the more specific we can be (e.g., *requiring* a linux version), the less likely that the field will be skipped or filled in with something that's not useful.

Maybe.

I can't argue with your logic, in general; however I still see value in a required pulldown for the basic universal environmental context with a text field for additional information.

But this doesn't have to be the final answer, and it's a vast improvement, so let's see how this works.